src/main/java/com/amazon/opendistroforelasticsearch/sql/query/maker/Maker.java

```diff
@@ -88,7 +89,7 @@
 
     private static final Set<Condition.OPEAR> NOT_OPEAR_SET = ImmutableSet.of(
             Condition.OPEAR.N, Condition.OPEAR.NIN, Condition.OPEAR.ISN, Condition.OPEAR.NBETWEEN,
-            Condition.OPEAR.NLIKE, Condition.OPEAR.NIN_TERMS, Condition.OPEAR.NTERM
+            Condition.OPEAR.NLIKE, Condition.OPEAR.NIN_TERMS, Condition.OPEAR.NTERM, OPEAR.NOT_EXISTS_NESTED_COMPLEX
```

Comment:
That's inconsistent with the rest, consider either change all to 
    
    OPEAR.N, OPEAR.NIN, ...

or add Condition.OPEAR.NOT_EXISTS_NESTED_COMPLEX

Reply:
Done.